docs/pages/academy/6-analytics/3-graph-analytics.md

```diff
@@ -15,120 +15,120 @@ At this point in the academy it should be somewhat clear that Grakn is not a gra
 
-Let's make an example: let us find the shortest path that links the company "ENI" to the country "UK" in our dataset. With the Grakn server running in your VM, run the following query in the dashboard
+An example: let us find the shortest path that links the company "ENI" to the country "UK" in our dataset. Run the following query in the dashboard
 
 ```graql-skip-test
 match $x isa company has name "ENI";
 $y isa country has name "UK"; get;
 ```
 
-This will bring up the two nodes of which you need to find the IDs. You could use one of the methods outlined above and paste the IDs somewhere, but there is a better way.
+Your result will contain two nodes holding their IDs. You could use one of the methods outlined above and copy the IDs somewhere, but there is a better way.
 
-Click on an empty point of the background of the graph visualiser to make sure that no node is selected. Then CTRL+Click anywhere on the background and you will see a green rectangle extending from the mouse pointer. You can use this window to select multiple nodes at the same time: just drag it over both the nodes currently shown as shown below and click again on the background. This will select both ENI and the UK.
+Click on an empty point of the background of the graph visualiser to make sure that no node is selected. Then CTRL+Click anywhere on the background and you will see a green rectangle extending from the mouse pointer. You can use this window to select multiple nodes at the same time: just drag it over both nodes currently shown as displayed below and click again on the background. This will select both ENI and the UK.
 
   ![Selecting multiple nodes](/images/academy/6-analytics/multi-selection.png)
 
-After this, you can right click one of the nodes and you will be able to select "Shortest path" among the options. If you click on it, the `compute path` query will be shown in the Graql editor ready for you to be launched or modified.
+After this, you can right click one of the nodes and you will be able to select the "Shortest path" ption. If you click on it, the `compute path` query will be shown in the Graql editor ready for you to be launched or modified.
 
   ![Shortest path GUI](/images/academy/6-analytics/path-gui.png)
 
 Runnning a path query in the Graph visualiser actually lets you see the path between the two nodes, as shown in the following screenshot:
 
   ![Shortest path in the Visualiser](/images/academy/6-analytics/path-visualiser.png)
 
-The result of the same query on the Dashboard console are a bit less interesting, but shows you how the result of the query looks like if you are interacting programmatically with the Grakn server.
+The result of the same query on the Dashboard console is a bit less interesting, but it is analogous to the result you would get from a programmatic interaction with the Grakn server.
 
   ![Shortest path in the Console](/images/academy/6-analytics/path-console.png)
 
-## Degrees
+## Centrality
```

Comment:
Here I think substituting Centrality for Degrees is a big change, I'll run it by someone who knows more than me before we add this, since it seems like a significant change in the terminology. This goes for the following section overall, I'll comment on each